src/frequenz/sdk/actor/_decorator.py

```diff
@@ -79,7 +79,7 @@ def actor(cls: Callable[_P, _R]) -> Type[_R]:
     Raises:
         TypeError: when the class doesn't have a `run` method as per spec.
 
-    Example (one actor receiving from two receivers):
+    Example (one actor receiving from two receivers): !this example is not updated to channels 0.16!
```

Comment:
To update it, you should probably only need to change the select loop to:

```py
        async def run(self) -> None:
            async for selected in select(self._recv1, self._recv2)
                if selected_from(selected, self._recv1):
                    await self._output.send(selected.value)
```

It is a pretty bad example still, but we'll address this when we revamp the actor decorator, so don't worry about it, just make sure the current example is minimally fixed.